tests/lib/collectors/azure_logging.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
```

Comment:
I'm a big fan of testing error cases, if it's possible to add some in.

Reply:
Done